modules/common/src/main/Licon.scala

```diff
@@ -135,3 +135,6 @@ object licon:
   val Move: Icon            = "" // e075
   val OneHalf: Icon         = "" // e076
   val Mute: Icon            = "" // e077
+  val Blindfold: Icon       = "" // e078
+  val AccountCircle: Icon   = "" // e079
+  val QuestionCircle: Icon  = "" // e07a
```

Comment:
why are `Blindfold` and `QuestionCircle` here? they don't seem to be used by the rest of the PR

Reply:
craplets from my master branch. i'll remove em later on when i add the logout link back and change button to an anchor.